### PR TITLE
completed css-pseudo-classes exercise - Bryan Brown

### PR DESCRIPTION
**Pull request by bbrown949**
Screenshot 2023-03-29 at 10 03 57 AM

### CODE REVIEW

File: css-pseudo-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+button {
+  height: 40px;
+  width: 150px;
+  font-size: 18px;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+}
```
Comment (shawnkost) [inline css-pseudo-classes/styles.css]: This should also be getting `cursor: pointer`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small change requested here Bryan!

File: css-pseudo-classes/styles.css (hunk 1 above)
Comment (bbrown949) [reply css-pseudo-classes/styles.css]: updated. Thanks.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**